publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -1746,6 +1747,54 @@ void publicationOwnerShouldBeAbleToUpdateMetadataOfPublishedFile() throws ApiGat
         assertThat(gatewayResponse.getStatusCode(), is(equalTo(HTTP_UNAUTHORIZED)));
     }
 
+    @Test
+    void userWhichHasAccessRightManageResourcesAllAndIsPartOfPublicationCuratingInstitutionsRepublishPublication()
+        throws ApiGatewayException,
+                                                                                       IOException {
+        var publication = TicketTestUtils.createPersistedPublication(PUBLISHED, resourceService);
+        var curatingInstitution = randomUri();
+        publication.setCuratingInstitutions(Set.of(curatingInstitution));
+        resourceService.unpublishPublication(publication);
+        var input = curatorWithAccessRightsRepublishedPublication(publication, randomUri(), curatingInstitution,
+                                                                  MANAGE_RESOURCES_ALL);
+
+        updatePublicationHandler.handleRequest(input, output, context);
+        var gatewayResponse = GatewayResponse.fromOutputStream(output, Publication.class);
+
+        var republishedPublication = resourceService.getPublication(publication);
+
+        assertThat(gatewayResponse.getStatusCode(), is(equalTo(HTTP_OK)));
+        assertThat(republishedPublication.getStatus(), is(equalTo(PUBLISHED)));
+    }
+
+    @Test
+    void shouldReturnForbiddenWhenRepublishingNotUnpublishedPublication()
```

Comment:
Not unpublished, or "published"? If "published" 200 OK is also OK since they get what they want